mne_bids/mne_bids.py

```diff
@@ -158,6 +158,55 @@ def _events_tsv(events, raw, fname, trial_type, verbose):
     return fname
 
 
+def _participants_tsv(fname, subject_id, age, sex, group, verbose):
```

Comment:
you don't have to pass `overwrite`?

Reply:
I didn't because it is the same issue as the scans file that I think I talked about a bit in #43 ...
We in general want the files to be overwritten, but the participants file we don't since it should be appended. If the participant is already in the file it won't be added again.
Essentially if `override = True` in the `raw_to_bids`, this may not be the intended behaviour on the participants file.

Reply:
I see. 

Continuing the discussion of #43, what do you think of changing the bool `overwrite` to `mode` and having three options for it `'overwrite`', `'append'` and `'error'`? This seems like the cleanest option to me.

Reply:
But okay -- for now, we can leave this as is